lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
         <a href="" class="nav-item is-active">My Teams</a>
-        <a href="" class="nav-item">Browse All Teams</a>
+        <a href="" class="nav-item">Browse</a>
       </nav>
 
       <div class="page-content-main">
-        <p>Blah blah blah. Go Team!</p>
+
+        <div class="search-tools">
+          <span class="search-count">Viewing 12 of 24 topics</span>
+          <span class="search-sort">sorted by
+          <select name="search-sort">
```

Comment:
@talbs I have a couple of concerns here.  How is the update triggered?  It cannot be onchange, it would have to be onblur or we need a submit button.  As far as notifying the user, we could either move focus to the card listing, or create an aria-live region and inject a text notification like "update complete", or both of those.